x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/index.test.tsx

```diff
@@ -76,9 +103,9 @@ describe('when on the hosts page', () => {
           });
 
           renderResult = render();
-          const detailsLink = await renderResult.findByTestId('hostnameCellLink');
-          if (detailsLink) {
-            reactTestingLibrary.fireEvent.click(detailsLink);
+          const hostNameLinks = await renderResult.findAllByTestId('hostnameCellLink');
```

Comment:
In the above "describe" string: maybe say:

> when the user clicks the first hostname in the table

That way the description will match. It might make it easier to understand future test failures

Reply:
True 😄 . Will change.